AppSearchSample/app/build.gradle

```diff
@@ -41,35 +42,33 @@ android {
         }
     }
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_17
+        targetCompatibility JavaVersion.VERSION_17
     }
     kotlinOptions {
-        jvmTarget = '1.8'
+        jvmTarget = '17'
     }
     buildFeatures {
         viewBinding true
     }
 }
 
 dependencies {
-    def appsearch_version = "1.0.0-alpha03"
+    kapt 'androidx.appsearch:appsearch-compiler:1.1.0-alpha03'
 
-    // AppSearch dependencies
-    implementation "androidx.appsearch:appsearch:$appsearch_version"
-    implementation "androidx.appsearch:appsearch-local-storage:$appsearch_version"
-    implementation "androidx.appsearch:appsearch-platform-storage:$appsearch_version"
-    kapt "androidx.appsearch:appsearch-compiler:$appsearch_version"
+    //api 'androidx.annotation:annotation:1.6.0'
```

Comment:
Not quite clear what this new comment is intended to communicate -- could you clarify?